IoTBlockly/SimpleWebServer/SimpleWebServer.cs

```diff
@@ -192,6 +193,7 @@ private Task WriteHeaderNoFlushAsync(Stream resp, int statusCode, string message
                 string header = String.Format(
                     "HTTP/1.1 {0} {1}\r\n" +
                     location +
+                    "Cache-Control:public, max-age=31536000\r\n" + // randon big number to ensure browser gets from cache dglover@microsoft.com 
```

Comment:
good change. no need to add your alias to the code. Git history will keep that info